sql/planbuilder/aggregates.go

```diff
@@ -164,6 +164,29 @@ func (b *Builder) buildGroupingCols(fromScope, projScope *scope, groupby ast.Gro
 	return groupings
 }
 
+func (b *Builder) buildNameConst(fromScope *scope, f *ast.FuncExpr) sql.Expression {
+	if len(f.Exprs) != 2 {
+		b.handleErr(fmt.Errorf("incorrect parameter count in the call to native function NAME_CONST"))
+	}
+	alias := b.selectExprToExpression(fromScope, f.Exprs[0])
+	aLit, ok := alias.(*expression.Literal)
+	if !ok {
+		b.handleErr(fmt.Errorf("incorrect arguments to: NAME_CONST"))
```

Comment:
errors with guidance instructions might be more helpful

Reply:
This error message is just copied from MySQL